src/components/Navbar/SettingsDrawer/WordByWordSection.tsx

```diff
@@ -209,6 +251,20 @@ const WordByWordSection = () => {
           />
         </div>
       </Section.Row>
+      <Section.Row>
+        <Section.Label>{t('fonts.font-size')}</Section.Label>
+        <Counter
+          count={wordByWordFontScale}
+          onIncrement={
+            MAXIMUM_WORD_BY_WORD_FONT_STEP === wordByWordFontScale
+              ? null
+              : onFontScaleIncreaseClicked
+          }
+          onDecrement={
+            MINIMUM_FONT_STEP === wordByWordFontScale ? null : onFontScaleDecreaseClicked
+          }
```

Comment:
Do you think we should put these checks in the callback function instead of doing it here?